.github/workflows/go.yml

```diff
@@ -35,9 +35,11 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
       - name: Install dependencies
-        run: go get -u github.com/jstemmer/go-junit-report
+        run: |
+          go get github.com/jstemmer/go-junit-report
+          go install github.com/jstemmer/go-junit-report
```

Comment:
How about this?

```suggestion
          go install github.com/jstemmer/go-junit-report@latest
```

Reply:
fixed
https://github.com/launchableinc/examples/pull/45/commits/482cc044c6532b08c322166269d71e91c98049c2